backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/EdgeComponentProps.java

```diff
@@ -56,4 +60,8 @@ public IEdgesRequestor getEdgesRequestor() {
     public DiagramRenderingCache getCache() {
         return this.cache;
     }
+
+    public MoveEvent getMoveEvent() {
+        return this.moveEvent;
+    }
```

Comment:
Same remark as above: this if this is optiona it should be made explicit using an `Optional<MoveEvent>`.

Reply:
done